packages/cli/generators/controller/index.js

```diff
@@ -18,6 +20,7 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
       type: 'controller',
       outdir: 'src/controllers/',
     };
+    debug(`artifactInfo: ${inspect(this.artifactInfo)}`);
```

Comment:
Can we wrap the debug statement as follows to avoid the parameter calculation if debug is not on?

if (debug.enabled) {
  debug(`artifactInfo: ${inspect(this.artifactInfo)}`);
}

Reply:
Done.